packages/expo-navigation-bar/package.json

```diff
@@ -36,8 +36,7 @@
   "dependencies": {
     "@react-native/normalize-colors": "0.81.0-rc.5",
     "debug": "^4.3.2",
-    "react-native-is-edge-to-edge": "^1.1.6",
-    "react-native-edge-to-edge": "1.6.0"
+    "react-native-is-edge-to-edge": "^1.1.6"
```

Comment:
You should bump it to `^1.2.1` to ensure that the installed version supports `edgeToEdgeEnabled`